vendor/engines/secure_rooms/app/models/secure_rooms/occupancy.rb

```diff
@@ -34,6 +36,10 @@ def orphan?
       orphaned_at?
     end
 
+    def orderable?
```

Comment:
I kind of feel like an occupancy shouldn't know or care if it's orderable. The OrderHandler should probably own that logic.